cmd/stork/stork.go

```diff
@@ -107,12 +108,17 @@ func main() {
 			Name:  "migration-admin-namespace",
 			Usage: "Namespace to be used by a cluster admin which can migrate all other namespaces (default: none)",
 		},
+		cli.BoolTFlag{
+			Name:  "cluster-controller",
```

Comment:
Should we call it storage controller? Cluster is too generic

Reply:
Storage controller assumes all storage operations.
I will call it storage-cluster-controller.